docs/admin/admin-ui-setup/dashboard-analysis.md

```diff
@@ -41,6 +41,21 @@ Toggle the switch to `disable` to disable *CPU Dashboards* option.
 
 The following analysis can apply to both GPU and CPU dashboards.
 
+### Total and Ready GPU or CPU Nodes
+
+The *Indicators* panel of the *GPU Overview Dashboard* displays the total number of GPU nodes, the number of ready GPU nodes, the total number of GPUs, and the total number of ready GPUs.
+
+The *Indicators* panel of the *CPU Overview Dashboard* displays the total number of CPU nodes, the number of ready CPU nodes, the total number of CPUs, and the total number of ready CPUs.
+
+These panes help calculate the number of available (unscheduled) resources in the platform.
+
+* **Total GPU/CPU Nodes**&mdash;indicates the number of nodes configured to work with the platform.
```

Comment:
I would use "sum total of nodes in all clusters connected to the platform" instead of "number of nodes configured to work with the platform